newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionParametersEditor.js

```diff
@@ -325,6 +370,16 @@ export default class EventsFunctionParametersEditor extends React.Component<
                               ),
                               click: () => this._removeLongDescription(i),
                             },
+                            {
+                              label: i18n._(t`Move up`),
+                              click: () => this._moveParameters(i, i - 1),
+                              enabled: i - 1 >= 0,
```

Comment:
Add also a check for `!isParameterDisabled(i)`: `!isParameterDisabled(i) && i - 1 >= 0`

Reply:
Actually, we should also check if the previous parameter is not disabled :) 
`!isParameterDisabled(i) && i - 1 >= 0 && !isParameterDisabled(i - 1)`